Decorator/CalculationService/CalculationServiceWithCorrection.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace namespaceEpam.NetMentoring.CalculationService
+{
+    public class CalculationServiceWithCorrection : CalculationServiceDecorator
+    {
+        private readonly int _corretion = 10;
```

Comment:
it's compile-time const - better to use  `const <type> Value`, also it can be change in .ctor - so what the idea of `readonly int `?

Reply:
ok